CONTRIBUTING.md

```diff
@@ -5,6 +5,18 @@ The most meaningful are:
 - Open a pull request
 - Vote
 - Give feedback
+---
+## Open a Pull Request
+If you want to add a feature or fix a bug. Pull Requests are the way to go. Try to keep them short and simple and follow the developer guide.  
+### Developer guide
+#### REQUIRED
+- Follow the [GDScript style guide](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#gdscript-style-guide)
+- Use [Scony"s godot gdscript tollkit](https://github.com/Scony/godot-gdscript-toolkit/) to validate the Pull Request
+#### RECOMMENDED
+- Folder names are in snake_case
+- File names are in snake_case
```

Comment:
Should be "Script file names are in snake_case"

Reply:
It doesn't distinguish between files by design.
True, file names are not covered by the GDScript style guide, but this does not hinder us from creating rules for file names.